torch/csrc/deploy/deploy.cpp

```diff
@@ -0,0 +1,110 @@
+#include <torch/csrc/deploy/deploy.h>
+
+#ifndef _WIN32
+#include <dlfcn.h>
+#include <libgen.h>
+#include <unistd.h>
+#endif
+
+extern "C" char _binary_libtorch_deployinterpreter_so_start[];
```

Comment:
perhaps a comment, about why this reference to the 'blank_library'?

Reply:
i figured out part of it so far-

these externs just mean - there are these symbols that point to the begin/end of a binary blob in memory

there is a 'blank_library' file, which i thought was more complicated than it was - i think it's just what it purports to be- a blank library.  it's purpose is so that these symbols are defined (but point to the same address, a bin blob of len 0) in the case of not building torch deploy.

I assume there is some other code that in the torch deploy case actually populates these symbols with the result of the torchdeploy.o object that is created with the  cmake custom command that does 
OUTPUT libtorch_deployinterpreter.o

Reply:
I've switched to building libtorch_deploy.so separately from libtorch_cpu.so for other reasons (adds huge size to the other one, _cpu has lots of build rules associated with it). This means we don't need blank library or windows ifdefs for deploy since deploy.cpp is only built conditionally. Added a comment for `_binary_libtorch_deployinterpreter_so_start`. It is built with the `ld -r -b binary`command.